docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
```

Comment:
i would move this below the next paragraph as it breaks the logic flow of the explanation

Reply:
Good idea. It makes it a lot clearer in that order.